### DISCUSSION TITLE
LL-HLS doesn`t work with native Android player

### DISCUSSION THREAD

**Discussion by mbakumenkov** [OvenMediaLabs/OvenMediaEngine#1717] [Help] [answered]
Hi guys! Thank you for a nice software!

I encountered a problem that LL-HLS stream doesn`t work on Android in the chrome native player (in Firefox it works, but seems like it standart HLS, not LL-HLS). Is there any way to solve this problem?

Thank you!

**Comment by getroot** [comment, marked as answer]
It is a limitation of the android native player. It is recommended to use a player such as OvenPlayer or Shaka Player.

**Comment by SeeMeLive-Scott** [comment]
What about media3 exoplayer?  Are there any issues with getting ll-hls working from oven media player to media3?
